### ISSUE TITLE
Backport to 1.19.2

### ISSUE THREAD

**Issue by itsdinkd** [Aizistral-Studios/No-Chat-Reports#401]
### Idea

You have fixed tons of issues with NoChatreports on mc 1.19.3, and 1.19.4 

However, majority of your userbase is still on 1.19.2 (Due to all major modpacks being on 1.19.2 and not the other .3 and .4 versions, except for "optimization client side modpacks"). 

**Please backport 2.X to 1.19.2 Fabric.**

### See downloads

Notice 1.19.2 with 51k, and all the other versions for higher MC versions do not come close.
![image](https://github.com/Aizistral-Studios/No-Chat-Reports/assets/35115304/f88a00b6-240c-4784-8afd-1bff0706f059)

**2.5m for 1.19.2** all higher versions do not come close. Most of your users are on 1.19.2
![image](https://github.com/Aizistral-Studios/No-Chat-Reports/assets/35115304/2f3f89be-2c0e-4ce1-9147-c42030ec830f)

**Comment by Madis0** [comment]
Keep in mind that chat signing on 1.19.2 [works very differently from 1.19.3 and up](https://youtu.be/48H5nMQ_8Yg), so not everything can be ported or it wouldn't make sense to do so in the same way. Maybe you should be more specific on what features would you even like a backport for?

**Comment by Aizistral** [comment]
I am aware that most of the users are still on 1.19.2. In fact, it seems that final 1.19.2 releases constitute the majority of mod's total downloads at this time. Due to this I am willing to make an exception and make a few updates for 1.19.2.

But as @Madis0 correctly stated, chat signing works very differently from 1.19.3 and onward, so most bugfixes and feature updates simply cannot be backported as is. I am open for suggestions on what should be backported and how.

**Comment by IlyaTheSawmill** [comment]
It seems like a good idea, but I always use Forge :)

**Comment by Aizistral** [comment]
> It seems like a good idea, but I always use Forge :)

Yeah, this would definitely not be specific to Fabric if I ever do it.
